code/game/machinery/computer/robot.dm

```diff
@@ -9,20 +9,25 @@
 	var/temp = null
 
 	light_color = LIGHT_COLOR_PURPLE
-	
+
 	var/safety = 1
 
 /obj/machinery/computer/robotics/attack_ai(var/mob/user as mob)
 	return src.attack_hand(user)
 
 /obj/machinery/computer/robotics/attack_hand(var/mob/user as mob)
-	if(..()) 
+	if(..())
 		return
-	if(stat & (NOPOWER|BROKEN)) 
+	if(stat & (NOPOWER|BROKEN))
 		return
-		
-	ui_interact(user)
-	
+	var/datum/game_mode/nations/mode = get_nations_mode()
+	if(!mode)
+		ui_interact(user)

```

Comment:
Having the gamemode delete all of these computers would be preferable to snowflaking it into their attack_hand checks.